openedx.yaml

```
@@ -0,0 +1,19 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: frontend-app-learner
+owner: mattcarter
```

Updated spec says that this should absorb the team info.
```suggestion
owner:
  type: team
  value: masters-devs
```